code/script.js

```diff
@@ -2,217 +2,277 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const filterButton = document.getElementById('filter')
+const winOrLose = document.getElementById('winOrLose')
+const winOrLoseText = document.getElementById('winOrLoseText')
+const playAgainButton = document.getElementById('playAgain')
+const winnerLoserImg = document.createElement('img');
+const winnerLoserP = document.createElement('p');
+const startBox = document.getElementById('startBox');
+const startButton = document.getElementById('startButton');
+const timerElement = document.getElementById('timer')
+const winOrLoseTimer = document.getElementById('winOrLoseTimer')
+
+//Adding classes to the DOM variables
+winnerLoserImg.className = 'winner-loser-img';
+winnerLoserP.className = 'winner-loser-p';
+//Setting children to the winOrLose element
+winOrLose.appendChild(winnerLoserImg);
+winOrLose.appendChild(winnerLoserP);
+
 
 // Array with all the characters, as objects
 const CHARACTERS = [
```

Comment:
I really do love that you changed the characters for your own, it makes the game so much more interesting to play! It's a big job to do this all manually. We wish we would have known how to do a fetch here right? 😅 Good job!